src/lib/content/projects/projectsOverviews.ts

```diff
@@ -1,5 +1,5 @@
-import type { ProjectOverview } from "../../interface/projectOverview.interface";
-import { ProjectCategoryEnum } from "$lib/interface/enums/project-cateogry-enum.enum";
+import type { ProjectOverview } from "../../types/projectOverview.interface";
+import { ProjectCategoryEnum } from "$lib/types/enums/project-cateogry-enum.enum";
```

Comment:
Only one change here!

Remember to follow the same naming convention for similar file types:
- "project-overview.interface.ts" and "project-cateogry.enum.ts"
or
- "projectOverview.interface.ts" and "projectCateogry.enum.ts"

Each one of those options is correct.
If you decide to name all TS typing files with camel case, be sure to maintain that convention.
Personally, I use the second option, the one with a hyphen ("-").

Also, be careful that in the enum file you have another error in naming: 'enum.enum' instead of 'enum.ts'